back/config.json

```diff
@@ -0,0 +1,10 @@
+{
+    "db": {
+        "name": "lms",
+        "user": "postgres",
+        "password": "postgres",
+        "host": "127.0.0.1",
+        "port": 5432,
```

Comment:
user, password, host, port should not be commited to the github repository but rather taken from environment variables, from 'dotenv' for example

Reply:
done